doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2131,7 +2131,7 @@ OCT-3174: Host on CIS gateway may throw assert if a connected headset is hard re
 
   **Affected platforms:** nRF5340 Audio DK
 
-.. rst-class:: v2-9-0-nRF54H20-rc1 v2-9-0 v2-8-0
+.. rst-class:: v2-9-0-nRF54H20-rc1 v2-8-0
```

Comment:
We have a PR open for KI - https://github.com/nrfconnect/sdk-nrf/pull/19745. I can add this change there. Then we could close this PR. Could you confirm do we need to remove the v2-9-0-nRF54H20-rc1 tag as well?